makes minor revs to ops setup + adds ssh install (#187)
codefellows/setup-guide: ops-setup/2-remote-access.md

# Ops Setup

## Configure Remote Access
Now, let's configure your lab so that you can access its Graphical User Interface (called a GUI) from your home computer over your home network.

> To do this we will first install a software package called `ssh`, which will give us access this Terminal on your lab computer from your home computer, allowing us to run commands just like are now.

Reviewer: ```suggestion
To do this we will first install a software package called `ssh`, which will give us access this Terminal on your lab computer from your home computer, allowing us to run commands just like we are now.
```

Second, we will install `xrdp`, which uses the Remote Desktop Protocol (often shortened to RDP), then set it to run automatically when the computer boots up.

Finally, we will configure the firewall to allow this network traffic.

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

### Install The `ssh` Software Package:
```bash
sudo apt install ssh -y
```

> > N.B. The `-y` option bypasses an "Are you sure?" prompt.

Reviewer: ```suggestion
> Note: The `-y` option bypasses an "Are you sure?" prompt.
```

### Install The `xrdp` Software Package:
```bash
sudo apt install xrdp -y
```

### Set `xrdp` To Run Automatically
This will tell the computer to run `xrdp` whenever the computer boots up, ensuring that when you reboot, you'll still be able to connect remotely.
```bash
sudo systemctl enable --now xrdp
```

> > N.B. `ssh` is enabled and started automatically when it is installed, so we do not need to enable it like this.

Reviewer: ```suggestion
> Note: `ssh` is enabled and started automatically when it is installed, so we do not need to enable it like this.
```
